lib/src/webcrypto/webcrypto.ecdh.dart

```diff
@@ -18,6 +18,41 @@ part of 'webcrypto.dart';
 abstract class EcdhPrivateKey {
   EcdhPrivateKey._(); // keep the constructor private.
 
+  /// Import [EcdhPrivateKey] in the [PKCS #8][1] format.
+  ///
+  /// The [keyData] parameter is an octet string whose 
+  /// contents are the value of the private key.
```

Comment:
What is keyData? 

See: https://www.w3.org/TR/WebCryptoAPI/#ecdh-operations > "Import Key" > If format is "pkcs8".

Or look at the export operation too.

-------------

Btw, looks like https://www.w3.org/TR/WebCryptoAPI/#ecdh-operations
refers to "privateKeyInfo ASN.1 structure defined in RFC 5280", but RFC 5208 is the one that defines privateKeyInfo, see:

https://www.ietf.org/rfc/rfc5208.txt

-------------------

The more I read I think the input is: `privateKeyInfo` structure.
Also reading: https://www.w3.org/TR/WebCryptoAPI/#concept-parse-an-asn1-structure

I get the impression that it has to be a DER encoded `privateKeyInfo` structure.

which is similar to what we stated in:
https://pub.dev/documentation/webcrypto/latest/webcrypto/RsassaPkcs1V15PrivateKey/importPkcs8Key.html

I don't think we have to specify that all the sub-properties of the `privateKeyInfo` must be.

But we might want to say something about the curve must match what is specified in ......

Reply:
Can we write something like - 
[keyData] is the DER encoding of the PrivateKeyInfo structure specified in [RFC 5208](https://tools.ietf.org/html/rfc5208). The [privateKeyAlgorithm] of the [keyData] must match with the [curve] specified in the parameter.

OR

The [keyData] should be generated for the same curve specified in the [curve] parameter.

Reply:
Yeah, we could write either!